src/Native/Runtime/CMakeLists.txt

```diff
@@ -135,12 +144,16 @@ else()
     set(ARCH_SOURCES_DIR i386)
   elseif(CLR_CMAKE_PLATFORM_ARCH_WASM)
     set(ARCH_SOURCES_DIR wasm)
+# Using C++ as our "assembly" language for WASM until we figure out how to compile WAST
```

Comment:
Same here - portable runtime flavor does not include any asm files already.

Reply:
Removed

Reply:
Removed